charts/rancher-turtles/templates/clusterctl-config.yaml

```diff
@@ -50,6 +50,6 @@ data:
 
     # Addon providers
     - name:         "fleet"
-      url:          "https://github.com/rancher-sandbox/cluster-api-addon-provider-fleet/releases/latest/addon-components.yaml"
+      url:          "https://github.com/rancher-sandbox/cluster-api-addon-provider-fleet/releases/v0.3.0/addon-components.yaml"
```

Comment:
are you sure that this has to be hardcoded? maybe it can be templated using config.yaml or Chart.yaml

Reply:
We can add that. Just wanted for now to get rid of `latest` since the feature is moved out of experimental and the version needs to be pinned. People can still install versions below `v0.3.0`, but it limits the upper bound or when the version is not specified at all. And this bound is shipped and controlled by each turtles release, so we know what to expect from new installs at all times.